af/analysis/rotation.py

```diff
@@ -64,9 +79,25 @@
 
 conffile_path = "/etc/ooni/rotation.conf"
 setup_script_path = "/etc/ooni/rotation_setup.sh"
+nginx_conf = "/etc/ooni/rotation_nginx_conf"
```

Comment:
Why not embed the nginx configuration content into this script itself? If not maybe it should be documented somewhere how this looks like.

Reply:
In order to store it in ansible.